### PR TITLE
PR 1 : Initial Version 

### PR DESCRIPTION
**Pull request by SAYUK09**

### CODE REVIEW

File: docs.css [hunk 1]
```diff
@@ -429,6 +429,18 @@ footer{
     border: 1px solid #3B82F6;
 }
 
+.iconBtn{
```
Comment (robinhooda) [inline docs.css:429]: It would be great if you add hover shadow to it

**Comment by robinhooda** [comment]
Really great job Sayuri !

File: docs.css (hunk 1 above)
Comment (SAYUK09) [reply docs.css:429]: Done ! Thanks

File: docs.html [hunk 2]
```diff
@@ -174,6 +173,78 @@ <h3>Copy and Use it ⬇</h3>
 
                 <hr>
 
+                <section id="iconButton">
+
+                    <h2>Icons Buttons</h2>
+
+                        <div class="componentDiv">
+
```
Comment (soham-parab) [inline docs.html:181]: I feel the design would look cleaner without the borders for each component.

File: docs.html [hunk 3]
```diff
@@ -268,11 +339,77 @@ <h3>Copy and Use it ⬇</h3>
...
+                        <div class="notifDiv">
+                            
+                            <ul class="list listNotif">
+                              <li class="notif ">Notification 1</li>
+                              <li class="notif ">Notification 2</li>
+                              <li class="notif ">Notification 3</li>
+                              <li class="notif ">Notification 4</li>
+                              <li class="notif ">Notification 5</li>
+                              <li class="notif ">Notification 6</li>
+                              <li class="notif ">Notification 7</li>
```
Comment (soham-parab) [inline docs.html:358]: I would suggest adding a scroll bar to the stacked list so it looks more like a notification tray.

File: docs.html [hunk 4]
```diff
@@ -268,11 +339,77 @@ <h3>Copy and Use it ⬇</h3>
...
+
+                        <div class="inputWithIcon">
+                            <i class="fa fa-envelope fa-lg fa-fw input" aria-hidden="true"></i>
+                            <input type="text" class="inputField" placeholder="Email">
+                        </div>
+
+                        <div class="inputWithIcon">
+                            <i class="fa fa-phone fa-lg fa-fw" aria-hidden="true"></i>
+                            <input type="text" class="inputField" placeholder="Phone Number">
+                        </div>
```
Comment (soham-parab) [inline docs.html:393]: I would suggest shortening the width of the input areas.

**Review by soham-parab — COMMENTED** [review]
Awesome job on the landing page and the components. Cards were nicely designed. Please consider the suggested changes, thank you.

File: docs.html (hunk 2 above)
Comment (SAYUK09) [reply docs.html:181]: Done ! looks much cleaner

**Pull request merged**